src/processors/feature.ts

```diff
@@ -129,6 +152,23 @@ export class FeatureFileProcessor extends TypeScriptFileProcessor {
         return;
       }
 
+      if (FeatureFileProcessor.isExportAssignment(node)) {
+        const { expression } = node;
+        if (FeatureFileProcessor.isObjectLiteralExpression(expression)) {
```

Comment:
```suggestion
        if (FeatureFileProcessor.isObjectLiteralExpression(expression) && expression.properties.length > 0) {
```

nit to guard against crashes for `export default {};`